test/envs/regional-dr.yaml

```diff
@@ -50,7 +50,6 @@ templates:
       - addons:
           - name: ocm-hub
           - name: ocm-controller
-          - name: cert-manager
```

Comment:
We have this addon in other environments, need to be removed from all.

Also we want to remove the addon. If there will be a need to use the addon in the future we can get it from earlier version. Maybe keep the addon removal in a separate commit to make it easy to revert if needed.

Reply:
done